src/features/viewport/zoom.ts

```diff
@@ -61,6 +61,37 @@ export class ZoomMouseListener extends MouseListener {
         return [];
     }
 
+    wheels(targets: SModelElement[], events: WheelEvent[]): Action[] {
```

Comment:
Same here, the wheel function may as well just call the wheels function now with single-element lists.

Reply:
also seems to be good